drbd/global_confs.sls

```diff
@@ -25,6 +25,9 @@
         quorum: "off"
         multi_primaries: "no"
         fencing: "resource-and-stonith"
+        max_buffers: 2048
```

Comment:
I guess these are the default values we should have above, and use here the values from `defaults`

Reply:
I leave it even not configured by default.

Reply:
The same on in `defaults.yaml` will be removed.